content/docs/command-reference/root.md

```diff
@@ -10,11 +10,11 @@ usage: dvc root [-h] [-q | -v]
 
 ## Description
 
-This command returns the path to the root directory of the <abbr>DVC
-project</abbr>, relative to the current working directory. It can be used to
-build a path to a dependency, script, or <abbr>data artifact</abbr>, for
-example. Useful when working in a subdirectory of the project, and needing to
-refer to a file in another directory.
+Returns the path to the root directory of the <abbr>DVC project</abbr>, relative
+to the current working directory. Useful when working in a subdirectory of the
```

Comment:
Sorry I'm going to be annoying and since I can't un-see your previous version, I think at least mentioning the command name once here would be nice:
```suggestion
to the current working directory. `dvc root` is useful when working in a
subdirectory of the ...
```
> Plus it makes it a complete sentence.

Reply:
Or maybe "`dvc repro` is useful to refer to a file in another directory when working in a subdirectory of the project."

Reply:
Agree with the change, but I'll wait on team comments before updating motivation.

Reply:
Adjusted this but used the `dvc root` reference in the second paragraph instead.